### DISCUSSION TITLE
Update documentation on custom building

### DISCUSSION THREAD

**Discussion by DirkO4YB** [HASwitchPlate/openHASP#521] [Documentation]
Hello all, I'm Dirk and this is my first 'lifesign' in the openHASP community. First let me say thanks to all the folks that have been working hard on this project, least of all mr. fvanroie. This looks very promising to me as i was looking for an open alternative to the Nextion screen for my home project. If i can get this all to work i will most certainly make a contribution. But firstly i will have to master the extra challenges, like using PlatformIO (until now i have only programmed my esp controllers on the Arduino platform). For my project  i need to do some customisation, like adding a temperature sensor to the Lanbon board. I am sad to say it allready took me 3 evenings of trial and error to get my first build run without problems. I thought it would be safest to start custum coding on a stable released version, the 0.6.3, like the documentation states as an option. After a lot of failed attempts, i only after 3 days found the statement from mr. vanroie that customisation on that version is broken and will not be worked on. I can see why that is, but it would be nice if this could be stated in the documentation. Just erase the option to customise on anything but the main branch (0.7.0rc..) This would have saved me some frustration... Yes ofcourse, i could have looked in the discussions earlier, but i tend to follow the documentation and try doing things as explained and only when i have made sure that i have followed the procedures correctly go look for known issues, and only after that search in discussions and communication groups. To summarise, please correct this in the documentation and help (elderly) newbees like me to take the first hurdles a bit easier. 
Thanks again, Dirk

**Comment by fvanroie** [comment]
Thanks Dirk. The documentation could certainly use some extra pair of eyes and contributions to make it better...
It's a tough job to cater to everyone's needs. 

The best way to improve the documentation is to contribute the changes you want to see directly to the https://github.com/HASwitchPlate/openHASP-docs repository. This way, if you find a mistake or want to add missing information, it's updated directly where it needs to go.

Many Hands Make Light Work.

**Comment by DirkO4YB** [comment]
Ahhh, i see. Normally, i'm a modest man, not the first to make changes to work that others have invested so much time and knowledge in. And i have never before worked in github including the doc service. However i know wiki's and i think i can pick up how it is done. I will take up your invitation and try to contribute here. I gather there is a role-back option for when i get it wrong, i hope you will supervise any modifications. Greetz, Dirk

**Reply by fvanroie** [reply]
Yes, of course the contributions are reviewed before merging and all modifications are tracked.
This in kind of the whole purpose of GitHub 😃
